reference/features/search_parameters.md

```diff
@@ -346,14 +346,12 @@ The highlighted version of the text would then be found in the `_formatted` obje
 **Expected value**: `true` or `false`
 **Default value**: `false`
 
-Adds an object to the search response (`_matchesInfo`) containing the location of each occurrence of queried terms across all fields. This is useful when you need more control than offered by our [built-in highlighting](#attributes-to-highlight).
+Adds a `_matchesInfo` object to the search response that contains the location of each occurrence of queried terms across all fields. This is useful when you need more control than offered by our [built-in highlighting](#attributes-to-highlight). `matches` only works for strings and numbers.
```

Comment:
Does it work only on fields containing strings or numbers or does it also work with values inside arrays and objects?

Reply:
It works for arrays of numbers and strings. Updated